inc/Modules/Rest/Search_Controller.php

```diff
@@ -49,6 +54,35 @@ public function register_routes(): void {
 				],
 			]
 		);
+
+		// Indexable entities (per site URL): get / set.
```

Comment:
Relocated from `basic-options`, used by both the governing site (js) and client sites (for indexing).

As previously discussed, may decide to combine things into a single endpoint in a _future_ pr.